share/locale/musescore_ast.ts

```diff
@@ -22853,13 +22853,13 @@ compás d&apos;anacrusa</translation>
         <location filename="../../src/instrumentsscene/view/layoutpaneltreemodel.cpp" line="888"/>
         <source>Are you sure you want to delete the selected %n instrument(s)?</source>
         <extracomment>Please omit `%n` in the translation in this case; it's only there so that you have the possibility to provide translations with the correct numerus form, i.e. to show &quot;instrument&quot; or &quot;instruments&quot; as appropriate.</extracomment>
-        <translation><numerusform>¿Tas seguru de querer desaniciar los %n instrumentos escoyíos?</numerusform><numerusform>¿Tas seguru de querer desaniciar l&apos;instrumentu escoyíu?</numerusform></translation>
+        <translation><numerusform>¿Tas seguru de querer desaniciar l&apos;instrumentu escoyíu?</numerusform><numerusform>¿Tas seguru de querer desaniciar los %n instrumentos escoyíos?</numerusform></translation>
```

Comment:
The addition of %n here seems not in line with the instructions in extracomment... same with the one below.

Reply:
True, removed

Reply:
oops, I forgot to save one change, done now